dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,344 @@
+package dbfit.environment;
+
+import dbfit.annotations.DatabaseEnvironment;
+import dbfit.api.AbstractDbEnvironment;
+import dbfit.util.DbParameterAccessor;
+import dbfit.util.Direction;
+import dbfit.util.NameNormaliser;
+import dbfit.util.Options;
+import dbfit.util.TypeNormaliserFactory;
+import fit.TypeAdapter;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+
+@DatabaseEnvironment(name="Informix", driver="com.informix.jdbc.IfxDriver")
+public class InformixEnvironment extends AbstractDbEnvironment  {
+
+    @Override
+    public void afterConnectionEstablished() throws SQLException {
+        TypeAdapter.registerParseDelegate(java.math.BigDecimal.class, new dbfit.util.BigDecimalParseDelegate());
```

Comment:
Looks like this `registerParseDelegate` call is already present in `FitNesseTestHost`. Do we need it here again?